.github/workflows/all-pr-tests.yml

```diff
@@ -4,6 +4,7 @@ on:
   pull_request:
     branches:
       - main
+      - feature/2.x
```

Comment:
Consider using `2.x` as a branch name here so you can target it in Composer for early testing (as `2.0`)

Reply:
ops, good catch. thanks.